pkg/server/plugin/upstreamauthority/disk/disk_test.go

```diff
@@ -332,7 +352,11 @@ func (s *DiskSuite) configureWithTTL(keyFilePath, certFilePath, deprecatedTTL st
 }
 
 func (s *DiskSuite) TestPublishJWTKey() {
-	resp, err := s.p.PublishJWTKey(context.Background(), &upstreamauthority.PublishJWTKeyRequest{})
+	stream, err := s.p.PublishJWTKey(context.Background(), &upstreamauthority.PublishJWTKeyRequest{})
+	s.Require().NoError(err)
+	s.Require().NotNil(stream)
+
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.